drm-format.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <graphics/graphics.h>
+#include <libdrm/drm_fourcc.h>
+
+#include "cef-headers.hpp"
+
+#ifdef ENABLE_BROWSER_SHARED_TEXTURE
+
+struct obs_cef_video_format {
+	cef_color_type_t cef_type;
+	uint32_t drm_format;
+	enum gs_color_format gs_format;
+	const char *pretty_name;
+};
+
+bool obs_cef_all_drm_formats_supported();
```

Comment:
I don't know if this is the same in C++, but in C this would have been:

```suggestion
bool obs_cef_all_drm_formats_supported(void);
```

Reply:
Not controversial so I made the change.